### ISSUE TITLE
Isn't the contributors guide needed on readme?

### ISSUE THREAD

**Issue by vikaskyadav** [collective/gatsby-starter-plone#8]

**Comment by vinay72** [comment]
Can I add the contributor guide in Readme.md file for new contributors to know how to contribute to this repository?

**Comment by ajayns** [comment]
Sure! Maybe @cekk or @tisto can help us with a repo with a good contributors guide which can be used as reference.

**Comment by vinay72** [comment]
@ajayns Can I add this file in this repository by sending a PR . You can check this link : [https://docs.plone.org/develop/coredev/docs/agreement.html](url)

**Comment by datakurre** [comment]
@vinay72 Just create a PR and discussion would be easier to handle in that PR.

Technically this package is not yet under Plone Foundation, so it does not require to follow those rules, but sure it would not do any harm to follow them.

**Comment by vinay72** [comment]
@datakurre Whenever I try to push my commit from my remote branch, I am getting the following error in my terminal 
![screenshot from 2019-02-11 01-13-30](https://user-images.githubusercontent.com/28500944/52538583-64bf8f80-2d9a-11e9-9903-bcaa1743f12f.png)

**Comment by datakurre** [comment]
Yes, you need to fork the repository and create fork from the pull. That’s common practise.

**Comment by ajayns** [comment]
Please read through the commonly used practices while contributing to open source and maybe work your way through Github before opening a proper PR. It doesn't make sense spamming this repo with PRs @vinay72

**Comment by vinay72** [comment]
@ajayns I am sorry for inconvenience.
